tools/cpp/windows_cc_configure.bzl

```diff
@@ -254,14 +254,17 @@ def find_vc_path(repository_ctx):
     auto_configure_warning_maybe(repository_ctx, "Visual C++ build tools found at %s" % vc_dir)
     return vc_dir
 
-def _is_vs_2017_or_2019(vc_path):
-    """Check if the installed VS version is Visual Studio 2017."""
+def _is_vs_2017_or_2019(repository_ctx, vc_path):
```

Comment:
Thanks! 

This check looks much better, according to https://devblogs.microsoft.com/cppblog/compiler-tools-layout-in-visual-studio-15/#where-the-compiler-toolset-moved-to-in-vs-2017, those three directory should always exist for 2017 and later version of Visual C++ Build Tools.

However, since file path is case-insensitive, can you convert all strings to lower case and then do the comparison? Just in case somehow the directory was created with different cases somehow.